src/vma/util/sys_vars.h

```diff
@@ -392,14 +392,24 @@ struct mce_sys_var {
 	sysctl_reader_t & sysctl_reader;
 
 private:
+	void get_env_params();
```

Comment:
Please change get_env_params() visibility to public function so it can be accessed from sock-redirect.cpp.